Public/Src/Cache/ContentStore/Distributed/NuCache/LocalLocationStoreConfiguration.cs

```diff
@@ -203,32 +205,62 @@ public DistributedCentralStoreConfiguration(AbsolutePath cacheRoot)
         public int MaxSimultaneousCopies { get; set; } = 10;
     }
 
+    /// <summary>
+    /// Provides Azure Storage authentication options for <see cref="BlobCentralStorage"/>
+    /// </summary>
+    public class AzureStorageCredentials
+    {
+        /// <nodoc />
+        public string ConnectionString { get; private set; } = null;

```

Comment:
Lets make it immutable without private setter, and you don't have to set it to null.